index.html

```diff
@@ -5,9 +5,22 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Gundam Store</title>
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP&family=Noto+Sans:wght@400;700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
+  <div class = "header">
+    <img id = "store-logo" src = "images/logo.png">
+    <h2 class = "navbar-icons">
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-shopping-cart" aria-hidden="true"></i>
+    </h2>
+  </div>
+  <h1>Products</h1>
+  <div class = "products-container">
```

Comment:
Avoid putting spaces between your attributes and the values.  It's just sort of odd.

Reply:
Whoops! I think my brain was on JavaScript assignment brain; I like having that spacing there. Very embarrassing though 🥴